src/controller/python/BUILD.gn

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
```

Comment:
Believe this should be the CHIP copyright

Reply:
Thanks for catching up. Just fixed. thanks